release-checklist.md

```diff
@@ -0,0 +1,42 @@
+## Release Checklist
+
+### Ruby Gem
+
+1. Ensure relevant pull-requests have been merged
+
+2. Update CHANGELOG to reflect all changes and set the release date
+("X.X.X - 20XX-XX-XX")
+
+3. Bump the VERSION of Sensu in `lib/sensu/constants.rb`
+
+4. Commit the CHANGELOG and VERSION changes ("major|minor|patch
+version bump, X.X.X")
+
+5. Release the "sensu" Ruby gem: `bundle exec rake release` (need gem
+signature private key)
+
+### Packages
+
+1. Create a [sensu-omnibus](https://github.com/sensu/sensu-omnibus)
+release tag using the Sensu Ruby gem version and a build iteration
```

Comment:
I think the description of the version tag needs a little more detail. How about adding something like this:

Each tag must have a "v" prefix, and join the  Sensu version with the build iteration with a dash (`-`).  For example, the second build iteration of Sensu version 1.0.1 would be tagged as `v1.0.1-2`.

Reply:
Added your content under the "Release Tagging" header.